src/gasman.h

```diff
@@ -1237,6 +1238,16 @@ Region *RegionBag(Bag bag);
 void *AllocateMemoryBlock(UInt size);
 #endif
 
+/*
+ * If not 0 this function will be called in
+ * CollectBags to allow users of libgap to mark bags
+ */
+extern TExtraMarkFuncBags ExtraMarkFuncBags;
+
+static void SetExtraMarkFuncBags(TExtraMarkFuncBags func)
+{ ExtraMarkFuncBags = func; }
```

Comment:
Same as above: Odd non-GAPish code formatting; and what's the point of a setter if wee don't hide the global variable anyway?